src/modules/ce/components/client/ClientReferralsPage.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import CommonTabs from '@/components/elements/CommonTabs';
+import PageTitle from '@/components/layout/PageTitle';
+import ClientOpportunitiesTable from '@/modules/ce/components/client/ClientOpportunitiesTable';
+import ClientReferralsTable from '@/modules/ce/components/client/ClientReferralsTable';
+
+const ClientReferralsPage: React.FC = () => {
+  return (
+    <>
+      <PageTitle title={'Referrals'} />
+
+      <CommonTabs
+        ariaLabel={'Client Referrals Tabs'}
+        tabDefinitions={[
+          {
+            title: 'All Referrals',
+            key: 'referrals',
```

Comment:
`clients/:id/referrals#referrals` is a little too weird, needs improvement.

I'm tempted to put `ce` in the URL (matching `projects/:id/ce`), even though I think we've been moving towards removing "ce" from user-facing language (e.g. "Referrals" instead of "Coordinated Entry" in the side nav item).

I'm proposing
`clients/:id/ce#referrals`
`clients/:id/ce#opportunities`

What do you think?

Reply:
I don't think we should make any changes to this yet, it feels too early to be fiddling with the routes. We have yet to really decide, UX-wise, whether we want to commit to language of "referrals" or to also call things "ce".